addons/project/static/tests/tours/project_tags_filter_tour_tests.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+
+function changeFilter(filterName) {
+    return [
+        {
+            trigger: '.o_favorite_menu button:has(i.fa-star)',
+            content: 'click on the favorite menu',
+        },
+        {
+            trigger: `.o_favorite_menu .dropdown-item span:contains("${filterName}")`,
+        },
+        {
+            trigger: '.o_group_by_menu button:has(i.oi-group)',
+            content: 'click on the groupby menu',
+            run: function (actions) {
+                this.$anchor[0].dispatchEvent(new Event('mouseenter'));
+            },
+        },
+        {
+            trigger: '.o_group_by_menu span:contains("Stage")',
+            content: 'click on the stage gb',
+        },
```

Comment:
Is the gb reset when selecting a filter?  :astonished:

Reply:
yep